### PR TITLE
Update Dutch & German Testimonials

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
+ simple_format

### CODE REVIEW

File: data/testimonials_de.yml [hunk 1]
```diff
@@ -30,63 +30,48 @@
...
-  summary: >
+  quote: |
+    Wir verglichen mehrere Softwareanbieter für Bildungsmanagement und Personalentwicklung. Defacto konnte sich mit seinen flexiblen Modulen in der Auswahl durchsetzen und überzeugt uns bis heute.
+
+    Die Kollegen von Defacto haben unseren Planungs- und Implementierungsprozess von CAPP Bilden sehr verlässlich und strukturiert begleitet und uns damit eine nutzerfreundliche Lösung geschaffen, die auf unser Haus zugeschnitten ist&#58; Das Importieren der Mitarbeiterdaten aus unserem HR-System ist erfolgreich, die Plattform wird intern gut genutzt und auch Externe können sich von unserem wachsenden Bildungskatalog in Anlehnung an unser Corporate Design überzeugen.
+
+    Die Anleitung und Beratung durch unsere festen Ansprechpartnerinnen ist stets kurzfristig möglich, lösungsorientiert und auf Augenhöhe. Mit der Einführung von <strong><a href="/capp-agile-learning/">CAPP Agile Learning</a></strong> hat Defacto eine passgenaue Lösung für Kunden geschaffen, in der uns ein breiter Anleitungskatalog und die neuesten Releasenotes zur Verfügung stehen. Damit sind wir stets auf dem neuesten Stand, können kurz Informationen nachschlagen und Neues erlernen.
```
Comment (fatboypunk) [inline data/testimonials_de.yml]: do we want a title or an alt on the `a`?

File: data/testimonials_de.yml [hunk 2]
```diff
@@ -30,63 +30,48 @@
...
+  quote: |
+    Im Máxima Medisch Centrum (MMC) brauchten wir ein System, welches das MMC-(Pflege)Personal stimuliert um kontinuierlich seine oder ihre Kompetenzen und Fähigkeiten einzusehen und weiter zu entwickeln, sodass die beste Pflege mit einem erkennbaren Profil geboten werden kann. Gemeinsam mit Defacto haben wir hierfür den Qualitätspass implementiert.
+
+    Schon vor der Implementierung des Qualitätspasses ist durch verschiedene Abteilungen, unter anderem der Personalabteilung und der Abteilung Qualität & Sicherheit, viel Vorarbeit geleistet worden in der Erstellung einer Basis. Wir haben uns für eine nachhaltige Lösung entschieden, wobei in Echtzeit die aktuellen Personaldaten und das daran gekoppelte Profil verfügbar sind.<br|Parallel baut der Qualitätspass auf eine Kultur auf, in der die Mitarbeiter selbst die Regie über ihr eigenes Expertenwissen übernehmen. Wichtig ist, dass Vorgesetzte und Abteilungsleiter eine Kultur unterstützen, in der man sich gegenseitig ansprechen kann und ansprechbar ist über das Folgen der Protokolle, sodass man gemeinsam Pflege von Spitzenqualität bietet.
```
Comment (fatboypunk) [inline data/testimonials_de.yml]: <br|Parallel

is this right or should it be <br>?

File: data/testimonials_de.yml [hunk 3]
```diff
@@ -30,63 +30,48 @@
...
-  quote: >
-    <p>
-      Das schönste an der Zusammenarbeit mit den Leuten von Defacto ist, dass wir schon so lange so einen intensiven Kontakt haben, dass man weiß das man aufeinander zählen kann. Dieses Vertrauen schafft auch Raum für kritische Anmerkungen. In den letzten Jahren haben wir so einen guten Kontakt aufgebaut, dass man das nicht so einfach mal zur Seite schiebt. Angenehm ist auch, dass sie an dem Ton der E-Mail oder Telefongespräch bereits merken ob etwas dringend ist oder nicht und immer für dich bereit stehen. Kurzum: eine ganz tolle Zusammenarbeit.
-    </p>
-  summary: >
+  quote: |
+    Das schönste an der Zusammenarbeit mit den Leuten von Defacto ist, dass wir schon so lange so einen intensiven Kontakt haben, dass man weiß das man aufeinander zählen kann. Dieses Vertrauen schafft auch Raum für kritische Anmerkungen. In den letzten Jahren haben wir so einen guten Kontakt aufgebaut, dass man das nicht so einfach mal zur Seite schiebt. Angenehm ist auch, dass sie an dem Ton der E-Mail oder Telefongespräch bereits merken ob etwas dringend ist oder nicht und immer für dich bereit stehen. Kurzum&#58; eine ganz tolle Zusammenarbeit.
```
Comment (fatboypunk) [inline data/testimonials_de.yml]: &#58 is a colon?

File: data/testimonials_nl.yml [hunk 4]
```diff
@@ -30,48 +30,36 @@
...
-    Heel prettig is het ook dat de mensen aan de toon van een mailtje of telefoontje al aanvoelen of iets dringend
-    is of niet en ze staan daarmee altijd voor je klaar. Kortom: een hele plezierige samenwerking.
-  summary: >
-    Het fijnste aan de samenwerking met de mensen van Defacto is dat we al zo lang zulk intensief contact hebben,
-    dat je weet dat je van elkaar op aan kunt. Door dit vertrouwen is er ook ruimte voor een kritische noot.
-    In de loop van de jaren hebben we zo’n goed contact opgebouwd dat je dat niet zomaar even aan de kant zet.
+  quote: |
+    Het fijnste aan de samenwerking met de mensen van Defacto is dat we al zo lang zulk intensief contact hebben, dat je weet dat je van elkaar op aan kunt. Door dit vertrouwen is er ook ruimte voor een kritische noot. In de loop van de jaren hebben we zo’n goed contact opgebouwd dat je dat niet zomaar even aan de kant zet.
+
+    Heel prettig is het ook dat de mensen aan de toon van een mailtje of telefoontje al aanvoelen of iets dringend is of niet en ze staan daarmee altijd voor je klaar. Kortom&#58; een hele plezierige samenwerking.
```
Comment (fatboypunk) [inline data/testimonials_nl.yml]: Same here with the &#58

File: data/testimonials_de.yml (hunk 3 above)
Comment (RonaldDijkstra) [inline data/testimonials_de.yml]: Yes

File: data/testimonials_de.yml (hunk 2 above)
Comment (RonaldDijkstra) [inline data/testimonials_de.yml]: It should be left out, simple_format takes care of it, but an extra enter doesn't hurt here.

File: data/testimonials_de.yml (hunk 1 above)
Comment (RonaldDijkstra) [inline data/testimonials_de.yml]: title

File: data/testimonials_de.yml (hunk 3 above)
Comment (fatboypunk) [inline data/testimonials_de.yml]: A colon doesn't work itself?

Comment (RonaldDijkstra) [inline data/testimonials_de.yml]: Yes works..

**Pull request merged**